genvm/templates/multisig/handler.go

```diff
@@ -86,23 +86,24 @@ func (h *handler) Parse(ctx *core.Context, method uint8, decoder *scale.Decoder)
 			return
 		}
 		args = &p.Arguments
-		header.GasPrice = p.GasPrice
-		header.MaxGas = h.totalGasSpawn
+		out.GasPrice = p.GasPrice
+		out.Nonce = p.Nonce
```

Comment:
this reminds me, where does the code check whether this account is already spawned?

Reply:
https://github.com/spacemeshos/go-spacemesh/blob/c7b97f0a2f586f303d4a27211841279da083c988/genvm/core/context.go#L47-L50